api/pkg/git/git.go

```diff
@@ -74,7 +74,14 @@ func (c *client) Fetch(spec FetchSpec) (Repo, error) {
 }
 
 func (c *client) initRepo(spec FetchSpec) (*LocalRepo, error) {
-	log := c.log.With("name", "repo").With("url", spec.URL)
+
+	cloneUrl := spec.URL
+
+	if spec.SSHUrl != "" {
```

Comment:
how would user used their credentials for private repo?
 this is just ssh url

Reply:
may be mounting volume at certain path 🤔 i guess

Reply:
? if yes?
wouldn't that work with http url?

Reply:
@sm43 yes user will have to volume mount their ssh keys.
>wouldn't that work with http url?

no